dev/ReplacerScript/BeanFactoryClassReplace.php

```diff
@@ -0,0 +1,408 @@
+<?php
+
+$moduleClassList = [
```

Comment:
Proof that I actually code-reviewed every file :P.

Reply:
Good spot haha and thanks for removing the temp script :)